### PR TITLE
README.md

### PR DESCRIPTION
**Pull request by nooob-developer**
This updates for README file for the project, making several improvements and corrections to the content.
Changes Made

Reorganized the sections to make the content more clear and easier to navigate
Made corrections to the installation instructions for Unix and Windows users
Added explanations for some of the more technical terms used in the README
Reformatted the PowerShell code examples using code blocks with the powershell language specifier for better 

readability

With these improvements and corrections, the project's README file has been enhanced and made more useful and readable for its users. By reviewing and implementing the necessary changes, users can easily install and run the project and make the most of its features.

Thanks to Nvpak team

### CODE REVIEW

**Review by Pakrohk — APPROVED** [review]
It seems that it can be useful. I am waiting for others to Review.

File: README.md [hunk 1]
```diff
@@ -64,40 +64,68 @@ Show
...
+If you have an idea for a new feature or improvement, feel free to suggest it and discuss it with the [NvPak](https://github.com/Pakrohk-DotFiles/NvPak) team.
 
-If the software plugins are not installed automatically after the first run, proceed as follows:
+Fork the NvPak repository to your own GitHub account.\
+Make your changes and commit them to your forked repository. Please make sure to follow the project's coding standards and best practices, and write clear and concise commit messages.
 
-1. Run the following command inside nvim:
+Submit a pull request from your forked repository to the main NvPak repository.\
+ Your changes will be reviewed by the NvPak team, who may provide feedback and request changes if necessary.\
+Once your changes are approved, they will be merged into the main NvPak repository and will be available to all users.\
```
Comment (lorem10) [inline README.md:123]: please delete the backslash

File: README.md [hunk 2]
```diff
@@ -64,40 +64,68 @@ Show
...
+
+
+# Contributing 🤝
+
 
-### Usage 🚀
+If you're interested in contributing to the project, we welcome your help in fixing bugs and adding new features.\
+ Here's how you can get started:
+ 
+Check the Projects section to see if there are any open issues or features that you'd like to work on. \
```
Comment (lorem10) [inline README.md:115]: It is better to link the project section

**Review by ARS101 — APPROVED** [review]
Changes are good enough
Awaiting for merge...

**Review by lorem10 — APPROVED** [review]

**Pull request merged**